src/Buerokratt.Common/Dmr/DmrService.cs

```diff
@@ -63,8 +69,24 @@ private static string EncodeBase64(string content)
             return base64;
         }
 
-        private static HttpRequestMessage CreateRequestMessage(DmrRequest request)
+        private async Task<Participant> ResolveDmr()
+        {
+            // Get Dmr Instance.
+            var dmrs = await _centOps.FetchParticipantsByType(ParticipantType.Dmr);
+
+            if (!dmrs.Any())
+            {
+                throw new KeyNotFoundException($"No active DMRs found.");
+            }
+
+            // Whilst the behaviour remains undefined - we'll return the first Dmr we receive.
+            return dmrs.First();
+        }
+
+        private async Task<HttpRequestMessage> CreateRequestMessage(DmrRequest request)
         {
+            var dmr = await this.ResolveDmr();
```

Comment:
You could probably drop the `this`.

```suggestion
            var dmr = await ResolveDmr();
```